packages/admin/resources/views/components/notification-manager.blade.php

```diff
@@ -4,10 +4,21 @@
         add: function (event) {
             this.notifications = this.notifications.concat(event.detail)
         },
+        addError: function (notificationObject) {
```

Comment:
All this code can be removed from the notification manager

Reply:
> About showing form errors as notifications, this is already possible without using JavaScript - #2906

Using the javascript version in notification.manager.blade.php would mean less overhead in backend.
The part with dispatching a browser event  from `InteractsWithForms`, could actually be removed entirely... the Tab field could easily be modified to look for errors in js instead.

Reply:
It's a breaking change if we now don't allow sending notifications for validation errors in the backend, and its more configurable this way. Also the notification manager is no longer used any more because we have a new notifications package, so please remove all of the notifications logic from this PR.